test/typecheck/missing-ref.txt

```diff
@@ -0,0 +1,26 @@
+;;; TOOL: wat2wasm
+;;; ARGS*: --enable-function-references
+;;; ERROR: 1
+(module
+  (func $f (param (ref $first_type))
+           (result (ref $second_type))
+    (local (ref $third_type))
+  )
+)
+(;; STDERR ;;;
+out/test/typecheck/missing-ref.txt:5:24: error: Unknown reference: $first_type
+  (func $f (param (ref $first_type))
+                       ^^^^^^^^^^^
+out/test/typecheck/missing-ref.txt:6:25: error: Unknown reference: $second_type
+           (result (ref $second_type))
+                        ^^^^^^^^^^^^
+out/test/typecheck/missing-ref.txt:7:17: error: Unknown reference: $third_type
+    (local (ref $third_type))
+                ^^^^^^^^^^^
+out/test/typecheck/missing-ref.txt:5:24: error: Unknown reference: $first_type
+  (func $f (param (ref $first_type))
+                       ^^^^^^^^^^^
+out/test/typecheck/missing-ref.txt:6:25: error: Unknown reference: $second_type
```

Comment:
Looking at the errors that come from `resolve-names.cc` It looks like this should be `undefined reference type variable \"%s\""`.. or something like that?

Reply:
I cahnged it to `undefined reference type name %s`